desktop/core/src/desktop/js/utils/html/deXSS.ts

```diff
@@ -20,6 +20,10 @@ const deXSS = (str?: undefined | boolean | string | number | null, options?: IOp
   if (str === null) {
     return 'null';
   }
+  //Fix for sanitize HTML returns empty string for boolean false values.
```

Comment:
Did this start to happen after the upgrade and was caught by the unit tests?

Reply:
Yes